### PR TITLE
add test dashbord views

### PR DESCRIPTION
**Pull request by cpalessio96**
### All Submissions:

*   [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
*   [x] Does your code has proper inline documentation? 

### Changes proposed in this Pull Request:

Unit test for dashboard views

### CODE REVIEW

**Comment by GiuseppeIuculano** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Coverage increased per file
===========================
+ includes/Dashboard/PaymentTemplate.php  58
+ includes/Dashboard/CancelPage.php  77
+ includes/Loader.php  15
         

Complexity increasing per file
==============================
- tests/test-dashboard.php  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=6105200&bid=19984441)

**Comment by GiuseppeIuculano** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- tests/test-dashboard.php  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=6105200&bid=19984441)

File: tests/test-dashboard.php [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+            (object) array(
+                'price'         => 5,
+                'taxAmount'     => 1,
+                'taxPercentage' => 10,
+            ),
+            (object) array(
+                'price'         => 4,
+                'taxAmount'     => 2,
+                'taxPercentage' => 20,
+            ),
```
Comment (HyprCoder) [inline tests/test-dashboard.php]: Create a real object instead of cast it.

**Review by HyprCoder — CHANGES_REQUESTED** [review]

**Review by HyprCoder — APPROVED** [review]

**Comment by GiuseppeIuculano** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- tests/test-dashboard.php  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=6105200&bid=19984441)

**Pull request merged**